yugabyte/src/yugabyte/ysql/client.clj

```diff
@@ -171,6 +170,29 @@
         #"(?i)Operation expired"
         {:type :fail, :error [:operation-expired m]}
 
+        ; Happens upon network partition,
+        ; usually invoked upon RPC request timeout
+        #"(?i)Timed out after deadline expired"
+        {:type :info, :error [:timeout m]}
+
+        ; Happens when tserver has been stopped,
+        ; invoked from PG backend via ProcessInterrupts as a part of CHECK_FOR_INTERRUPTS macro
+        #"(?i)Terminating connection due to administrator command"
+        {:type :fail, :error [:conn-closed m]}
+
+        ;
+        ; PG driver-level errors
+        ; (usually results in operation failure, but we can't guarantee that)
+        ;
+
+        ; Might happen on basically any stage
+        #"(?i)This connection has been closed"
+        {:type :info, :error [:conn-closed m]}
+
+        ; Happens when there's a problem communicating with server
```

Comment:
which server is meant here?

Reply:
Given that this is `PG driver-level errors` section, PG driver only communicates with tserver

Reply:
Then it is worth to make it clear in section description.